tests/test_fft.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+import dpnp
+
+import numpy
+
+@pytest.mark.parametrize("type", ['complex128', 'complex64', 'float32', 'float64', 'int32', 'int64'])
+def test_fft(type):
+    # 1 dim array
+    data = numpy.arange(100, dtype=numpy.dtype(type))
+    # TODO:
+    # doesn't work correct with `complex64` (not supported)
+    # dpnp_data = dpnp.arange(100, dtype=dpnp.dtype(type))
+    dpnp_data = dpnp.array(data)

```

Comment:
```suggestion
    dpnp_data = dpnp.arange(100, dtype=numpy.dtype(type))

```
it would be better to use same creation procedure for both (expected and test data). To avoid issues in other function which are not related to this test.

Reply:
I have had dilemma for `complex64`, see comment below. But in any way I agree with you, actually `complex64` are disabled now. So I will change.